public/app/features/alerting/unified/components/receivers/form/ChannelSubForm.tsx

```diff
@@ -90,12 +90,12 @@ export function ChannelSubForm<R extends ChannelValues>({
     return () => subscription.unsubscribe();
   }, [selectedType, initialValues, setValue, fieldName, watch]);
 
-  const [_secureFields, setSecureFields] = useState(secureFields ?? {});
+  const [_secureFields, setSecureFields] = useState<Record<string, boolean | ''>>(secureFields ?? {});
 
   const onResetSecureField = (key: string) => {
     if (_secureFields[key]) {
       const updatedSecureFields = { ..._secureFields };
-      delete updatedSecureFields[key];
+      updatedSecureFields[key] = '';
```

Comment:
deleting the field, we were omitting this field in the request, making BE thinking that the field has not been updated.